Documentation/building/cross-building.md

```diff
@@ -91,4 +91,22 @@ The output is at bin/<BuildOS>.AnyCPU.Debug.
     drwx------  33 lgs lgs  4096  3  6 21:33 Windows_NT.AnyCPU.Debug
     drwx------  33 lgs lgs  4096  3  6 19:36 Windows_NT.AnyCPU.Release
 
+Building corefx for Linux ARM Emulator
+=======================================
 
+It is possible to build corefx binaries (native and managed) for the Linux ARM Emulator (latest version provided here: [#3805](https://github.com/dotnet/coreclr/issues/3805)).
+The `scripts/arm32_ci_script.sh` script does this.
```

Comment:
I thought that the script was also located in /dotnet/setenv of the mounted emulator rootfs. You may keep the existing statement without any modification.